### PR TITLE
Porfolio :Finish Mobile Version

### PR DESCRIPTION
**Pull request by NoelLincoln**
Add works section and about myself section
Use grid layout to place card elements in a colum and flexbox to place elements within the card element
Use JQuery toggle function on dropdown list to show and hide languages

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -71,8 +74,289 @@ <h1>I’m Noel Bryant from Kenya. Glad to see you!</h1>
...
+
+            <ul class="dropdown-content-skills" role="menu">
+              <li><a href="#">React</a></li>
+              <li><a href="#">Angular</a></li>
+              <li><a href="#">Vue</a></li>
+              <li><a href="#">Svelte</a></li>
+            </ul>
+          </div>
+        </div>
+
```
Comment (Efiamotu-1) [inline index.html:350]: - [ ] Your technology categories such as `languages, frameworks and skills` should be wrapped in `ul` and `li` tags. It is important to make use of semantic list item tags to display the list of technologies. :+1:

File: index.html
Comment (Efiamotu-1) [inline index.html:1]: - [ ] I want to kindly suggest that you add the button interactions on all buttons such as `hover, active` as provided by the figma template below. It is important to add interactivity to your web page as it makes it more functional and interactive to users who visit it :+1: :sparkles: 
- button interaction
b-i

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @NoelLincoln , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DuaneDave — APPROVED** [review]
Hi @NoelLincoln 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ All issues completely addressed
✔️ Good job adding the dropdown functionality, so awesome

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**